### PR TITLE
Fixes Issue #244

### PR DESCRIPTION
**Pull request by nakul-shahdadpuri**
Regarding [Issue](https://github.com/p5py/p5/issues/244)
Added  Synchronous versions of **http_get()** and **http_post()**
Please let me know about any changes, and where should I update the documentation and examples.
Thank you. :)

### CODE REVIEW

**Comment by ziyaointl** [comment]
The changes look great!

For the documentation, we use sphinx and the readthedocs theme. To get started, install some denpendencies:
```bash
pip install sphinx sphinx_rtd_theme
```
Then you should be able to run `make html` inside the docs folder. The outputs will be in _build/html/
You can start with adding the function signatures and their descriptions in the docs/reference folder by copying an existing file and renaming it to io.rst.

Please let me know if you have any questions!

**Comment by nakul-shahdadpuri** [comment]
@ziyaointl 
I have updated the docs for the added functions.
Let me know about any issues.
Thank you. :)

File: p5/io/http.py [hunk 1]
```diff
@@ -15,6 +22,15 @@ def http_get(url):
 
 #synchronous
 def http_post(url,post_data):
+	"""
+	Performs a synchronous POST request on the given url.
+
+	:param url: url of the API
+	:type url: string
+	:param post_data: Key-value pairs of the request payload.
+	:type post_data: map
```
Comment (ziyaointl) [inline p5/io/http.py:31]: Maybe this should be a `dict` since it's in Python?

**Comment by ziyaointl** [comment]
Thanks for updating the docs! This is awsome.

File: p5/io/http.py (hunk 1 above)
Comment (nakul-shahdadpuri) [reply p5/io/http.py]: Ok I ll make the change. :)

**Pull request merged**